### DISCUSSION TITLE
System.Runtime.InteropServices.COMException: '(41,4):UserId:'

### DISCUSSION THREAD

**Discussion by curlydonkey** [dahall/TaskScheduler#875] [Q&A]
I am attempting to add a task on a remote machine but the following exception is being raised when I call TaskService.AddTask():

`System.Runtime.InteropServices.COMException
  HResult=0x80070534
  Message=(41,4):UserId:
  Source=<Cannot evaluate the exception source>
  StackTrace:
<Cannot evaluate the exception stack trace>`

My code:

`var taskService = new TaskService("192.168.129.62", "xxx", "xxx", "xxx");
taskService.AddTask("xxx", QuickTriggerType.Daily, "myprogram.exe");`

I have checked and double-checked the credentials I am using for the remote machine.

**Comment by dahall** [comment]
Check [here](https://github.com/dahall/TaskScheduler/wiki/TaskSecurity) and search the other closed issues. This is common question with many responses that should cover your scenario.
